docs/rst/manual/keywords/index.rst

```diff
@@ -1367,6 +1383,42 @@ See the sub keyword :code:`OVERLAP_LIMIT` under the :code:`UPDATE_SETTINGS`keywo
 	A summary of the data used for updates are stored in this directory.
 
 
+.. _update_settings:
+.. topic:: UPDATE_SETTINGS
+
+        The :code:`UPDATE_SETTINGS` keyword is a *super-keyword* which can be used to
+	control parameters which apply to the Ensemble Smoother update algorithm. The
+	:code:`UPDATE_SETTINGS`currently supports the two subkeywords:
+
+   	OVERLAP_LIMIT Scaling factor used when detecting outliers. Increasing this
+        factor means that more observations will potentially be included in the
+        assimilation. The default value is 3.00..
+
+	Including outliers in the Smoother algorithm can dramatically increase the
+	coupling between the ensemble members. It is therefore important to filter out
+	these outlier data prior to data assimilation. An observation, \textstyle
+	d^o_i, will be classified as an outlier if
+
+	::
```

Comment:
The equation and the `\textstyle` does not render nicely to html. Try to use the rst-keywords 
`:math:` and `.. math::` (see https://shimizukawa-sphinx.readthedocs.io/en/latest/ext/math.html for more information) instead.

Also, while looking at the content it is a bit confusing when the outlier constraint applies as the text mostly refers to avarages, while the equation is indexed.